packages/ripple-nuxt-tide/lib/core/middleware.js

```diff
@@ -4,9 +4,10 @@ import { isPreviewPath } from '../../modules/authenticated-content/lib/preview'
 import logger from './logger'
 
 // Fetch page data from Tide API by current path
-export default async function (context, results) {
-  results.tidePage = null
-  results.tideErrorType = null
+export default async function (context, pageData) {
+  pageData.tidePage = null
```

Comment:
I renamed `results` to `pageData` for middleware as I have done in previous middleware PR, I think it more make sense to me.